### DISCUSSION TITLE
关于 flatbuffers 的交叉编译

### DISCUSSION THREAD

**Discussion by fasiondog** [xmake-io/xmake#3659] [General]
像 flatbuffers 这种在进行交叉编译时，如何同时生成主机平台的 flatc，以便可以调用 flatc 生成相应的源码文件？

**Comment by waruqi** [comment]
参考下 https://github.com/xmake-io/xmake-repo/blob/master/packages/l/luajit/port/xmake.lua

先生成host bin，然后生成代码参与后续编译

**Reply by fasiondog** [reply]
这个 luajit 用的 xmake 的 rules 机制，那像 flatbuffers 这种用 cmake 的，有简单的方法吗？

**Reply by waruqi** [reply]
都用 cmake 了。直接走 binary 包不就好了。
